python/tvm/relay/op/contrib/tensorrt.py

```diff
@@ -1033,6 +1033,8 @@ def visit_tuple_getitem(self, op):
         visit = super().visit_tuple_getitem(op)
         if visit.index != 0:
             return visit
+        if isinstance(visit.tuple_value, Call) and isinstance(visit.tuple_value.op, Let):
+            return visit
```

Comment:
This looks a bit ad-hoc to me. Suggest as follows. Note that I'm not sure if `isinstance(visit.tuple_value.op, Op)` is the correct expression but you should know what I meant.

```
if isinstance(visit.tuple_value, Call):
  if isinstance(visit.tuple_value.op, Op) and visit.tuple_value.op.name == "nn.dropout" and visit.index == 0:
    return visit.tuple_value.args[0]
return visit
```

Reply:
Thanks @comaniac . Changed the logic as appropriate.